src/test/java/chatBot/bot/ParserTest.java

```diff
@@ -0,0 +1,36 @@
+package chatBot.bot;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Should this be split into the different assertions you need to import instead of using the wildcard import?